README.md

```diff
@@ -60,7 +60,7 @@ Tool actions for your workflow.
 
 <!--lint ignore awesome-spell-check-->
 
-- [actions/github](https://github.com/actions/github) - Wraps actions-toolkit into an Action for common GitHub automations.
+- [actions/github-script](https://github.com/actions/github-script) - Wraps [octokit/rest.js](https://github.com/octokit/rest.js) into an Action for common GitHub automations.
```

Comment:
```suggestion

```
The `actions/github-script` already exists. You can just delete `actions/github`.

Reply:
Done.